src/libs/actions/Policy/Category.ts

```diff
@@ -334,6 +334,66 @@ function renamePolicyCategory(policyID: string, policyCategory: {oldName: string
     API.write(WRITE_COMMANDS.RENAME_WORKSPACE_CATEGORY, parameters, onyxData);
 }
 
+function setPolicyCategoryGLCode(policyID: string, categoryName: string, glCode: string) {
+    const policyCategoryToUpdate = allPolicyCategories?.[`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyID}`]?.[categoryName] ?? {};
+
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyID}`,
+                value: {
+                    [categoryName]: {
+                        ...policyCategoryToUpdate,
+                        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                        pendingFields: {
+                            glCode: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                        },
```

Comment:
IMO we won't need pending stuff, I think this would be offline pattern A, offline no feedback - cuz this is such a tiny field to update, but would be good to confirm in the doc

Reply:
hmmmm - `Does the user need to know if the action was successful?`

I think yes because the user would go ahead and export CSVs thinking glCode thing works but would get frustrated that it didn't work?? 

<img width="423" alt="image" src="https://github.com/Expensify/App/assets/29673073/e1369116-e1fa-44ca-a259-b10e4a1bfb14">

Reply:
I like how there's 2 "Most preferred" options hahaha. Yeah 🤷  IMO it's going to be insanely unlikely that these DON'T get updated successfully and IMO I'd prefer pattern A - cleaner code. But let's see what happens in the doc :D

Reply:
Resolving since we're going with pattern B!